test/powershell/Modules/Microsoft.PowerShell.Management/PSDrive.Tests.ps1

```diff
@@ -0,0 +1,41 @@
+Describe "Validate basic PSDrive Cmdlets" -Tags "CI" {
+    BeforeAll {
+        #just use same location as TestDrive for simplicity
+        $psDriveRoot = "TestDrive:"
+        $psDriveName = "PsTestDriveName"
+    }
+
+    BeforeEach {
+        New-PSDrive -Name $psDriveName -PSProvider FileSystem -Root $psDriveRoot
+    }
+
+    AfterEach {
+        Remove-PSDrive -Name $psDriveName -ErrorAction SilentlyContinue
+    }
+
+    It "Create a new PSDrive" {
+        $newDrive = New-PSDrive -Name "NewDifferentPSDrive" -PSProvider FileSystem -Root $psDriveRoot
+        $newDrive.Name | Should Be "NewDifferentPSDrive"
+        $newDrive.Root | Should Be (Convert-Path $psDriveRoot)
+        Remove-PSDrive -Name "NewDifferentPSDrive" -ErrorAction SilentlyContinue
```

Comment:
`Remove-PSDrive` should be put in a finally block, so even if the above `Should Be` failed, the test drive can still be cleaned up.